.github/workflows/ci-orchestrator.yml

```diff
@@ -0,0 +1,56 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+name: "CI Workflow Orchestrator"
+
+on:
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - ready_for_review
+      # Synchronize represents any kind of push or change to head ref on an open PR.
+      - synchronize
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  ci-orchestrator:
+    if: ${{ !github.event.pull_request.draft }}  # Don't run in a draft.
+    runs-on: ubuntu-latest
+    strategy:
+      max-parallel: 8  # Arbitrary - change or remove.
```

Comment:
As many as we can!

Reply:
Haha yeah absolutely. I added this just to help remind me this field exists. I'll definitely push it to "as many as we can" once it's been tested.